src/config/database.ts

```
@@ -3,6 +3,10 @@ import { withAccelerate } from "@prisma/extension-accelerate";
 import { config } from "./env";
 import { logger } from "./logger";
 import { trace, SpanStatusCode } from "@opentelemetry/api";
+import {
+  poolAcquireHistogram,
+  poolExhaustedCounter,
+} from "./promMetrics";
```

ESLint

[error] Replace `⏎··poolAcquireHistogram,⏎··poolExhaustedCounter,⏎` with `·poolAcquireHistogram,·poolExhaustedCounter·`

(prettier/prettier)